assets/inbuilt/transformers/dockerfile/net5app/templates/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#   Copyright IBM Corporation 2020
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+FROM mcr.microsoft.com/dotnet/sdk:5.0 AS buildImg
+WORKDIR /src
+COPY . .
+RUN mkdir app
+RUN dotnet restore {{ .CsprojFilePath }}
+RUN dotnet build {{ .CsprojFilePath }} -c Release -o /src/app
+RUN dotnet publish {{ .CsprojFilePath }} -c Release -o /src/app
+
+# Run Stage
+FROM mcr.microsoft.com/dotnet/aspnet:5.0 AS runImg
```

Comment:
Do we need to name this image?

Reply:
No, we don't need to name this image. I have removed it.